src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -1,164 +1,240 @@
-                 i18n-heading>
-              <span *ngFor="let line of contentData.audit_log">
-                {{ line.stamp }}&nbsp;{{ line.priority }}&nbsp;
-                <span [ngStyle]="line | logColor">
-                  <span style="font-weight: bold;">
-                    {{ line.message }}
-                  </span>
-                  <br>
-                </span>
-              </span>
-            </tab>
-          </tabset>
-        </fieldset>
+      {{ contentData.pools.length }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Raw Capacity"
+                  class="col-5"
+                  cardClass="card-medium"
+                  contentClass="card-chart"
+                  *ngIf="contentData.mon_status">
+      <cd-health-pie [data]="contentData"
+                     [isBytesData]="true"
+                     [displayLegend]="true"
+                     (prepareFn)="prepareRawUsage($event[0], $event[1])">
+      </cd-health-pie>
+    </cd-info-card>
+
+    <cd-info-card cardTitle="Objects"
+                  class="col-5"
+                  cardClass="card-medium"
+                  contentClass="content-medium content-highlight"
+                  *ngIf="contentData.df?.stats?.total_objects">
+      {{ contentData.df?.stats?.total_objects }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="PGs per OSD"
+                  class="col-5"
+                  cardClass="card-medium"
+                  contentClass="content-medium content-highlight"
+                  *ngIf="contentData.pg_info">
+      {{ contentData.pg_info.pgs_per_osd | round }}
+    </cd-info-card>
+
+    <cd-info-card cardTitle="PG Status"
+                  class="col-5"
+                  cardClass="card-medium"
+                  contentClass="card-chart"
+                  (click)="pgStatusTarget.toggle()">
+      <ng-template #pgStatus>
+        <p class="logs-link">
+          &rarr; See <a (click)="viewportScroller.scrollToAnchor('logs')">Logs</a> for more details.
+        </p>
+        <ul>
+          <li *ngFor="let pgStatesText of contentData.pg_info.statuses | keyvalue">
+            {{ pgStatesText.key }}: {{ pgStatesText.value }}
+          </li>
+        </ul>
+      </ng-template>
+      <div class="pg-status-popover-wrapper">
+        <span [popover]="pgStatus"
+              triggers=""
+              #pgStatusTarget="bs-popover"
+              placement="bottom">
+          <cd-health-pie [data]="contentData"
+                         chartType="pie"
+                         [displayLegend]="true"
+                         (prepareFn)="preparePgStatus($event[0], $event[1])">
+          </cd-health-pie>
+        </span>
       </div>
-    </div>
-  </div>
+    </cd-info-card>
+  </cd-info-group>
+
+  <cd-info-group groupTitle="Logs"
+                 class="row info-group"
+                 id="logs">
+
+    <cd-info-card cardTitle="Cluster"
+                  class="col-md-6"
+                  cardClass="card-medium"
```

Comment:
Can we set the logs card a bit bigger? we can only see 4 lines atm.
And unset the justified text.

Reply:
Logs card a bit bigger: Ack.
The justified text was requested by someone, I don't remember who.
As this is very subjective, can we reach a consensus about being or not justified,
because another one can ask to put it again...

Reply:
Done.